modBoot/src/org/aion/Aion.java

```diff
@@ -74,31 +74,41 @@ public static void main(String args[]) throws InterruptedException {
         if(cfg.fromXML())
             cfg.toXML(new String[]{ "--id=" + cfg.getId() });
 
-        
+
         try {
             ServiceLoader.load(AionLoggerFactory.class);
         } catch (Exception e) {
             System.out.println("load AionLoggerFactory service fail!" + e.toString());
             throw e;
         }
 
-
-        // If commit this out, the config setting will be ignore. all log module been set to "INFO" Level
-        AionLoggerFactory.init(cfg.getLog().getModules());
-        Logger LOG = AionLoggerFactory.getLogger(LogEnum.GEN.toString());
-
-        System.out.println(                
-                        "                     _____                  \n" +
+        System.out.println(
+                "                     _____                  \n" +
                         "      .'.       |  .~     ~.  |..          |\n" +
                         "    .'   `.     | |         | |  ``..      |\n" +
                         "  .''''''''`.   | |         | |      ``..  |\n" +
                         ".'           `. |  `._____.'  |          ``|\n\n" +
                         "                    NETWORK  v" + KERNEL_VERSION +
-                                "\n\n"
-                );
+                        "\n\n"
+        );
+
+        /** Outputs relevant logger configuration */
+        if (!cfg.getLog().getLogFile()) {
+            System.out.println("Logger disabled; to enable please check log settings in config.xml\n");
+        } else if (!cfg.getLog().isValidPath() && cfg.getLog().getLogFile()) {
+            System.out.println("File path is invalid; please check log setting in config.xml\n");
+            return;
+        } else if (cfg.getLog().isValidPath() && cfg.getLog().getLogFile()) {
+            System.out.println("Logger file path: '" + cfg.getLog().getLogPath() + "'\n");
+        }
+
+        // If commit this out, the config setting will be ignore. all log module been set to "INFO" Level
```

Comment:
not clear what that comment is mean to say

Reply:
Removed comment